### ISSUE TITLE
Add template reository commands

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-cli#66]
`metanorma template source add` adds template repo to metanorma configuration. 

`metanorma template source list` What sources are available. 

`metanorma template source remove` 

Or change `source` to another word.

**Comment by abunashir** [comment]
@ronaldtse: Why do we need that many levels of nested commands, what do you think about making it bit simpler, something like?

```
metanorma template

# add teamplte
metanorma template add

# list template
metanorma template list
....
```

**Comment by ronaldtse** [comment]
There are two separate concepts:

1. Template repository -- contains many templates
2. Template -- a single template

**Comment by abunashir** [comment]
yes, but I don't understand the purpose of having it nested. How does that help? maybe you wanna explain the whole plan with template interface?

**Comment by ronaldtse** [comment]
It doesn't have to be nested. As long as the user can manage template repositories and list templates, that's fine. Do you have any examples?

**Comment by abunashir** [comment]
cool, I'm working on the `template add` interface, should push that one soon.
